### DISCUSSION TITLE
Adding a link "forgot your passeword?" on the login screen

### DISCUSSION THREAD

**Discussion by AlexD-Bearing** [openkfw/TruBudget#854] [Ideas]
### Description
On user login page, create a "forgot your password?"   functionality, that allows to automatically reset the password via the email   adress that has been linked to the account.

### TODO
- [ ] ui: Add "forgot your password?" below the password field @ login screen
- [ ] ui: Create a password change page (similar style to login page)
             Password change fields:
                       - Email address

- [ ] ui: Disable the button if email service is not available/ create tooltip which informs about the missing email service
- [ ] ui/api: Check if user is disabled
How to check this without auth token?
- [ ] ui/email-service: Check if email exists
Create own endpoint without auth token?
- [ ] ui/email-service: Send restore password email (link)
How to create the link?

**Comment by mushtaq96** [comment]
Hello is this till open?
